### PR TITLE
Fix for "L&F breaks tab navigation #114"

### PR DESCRIPTION
**Pull request by markiewb**
- by invoking org.netbeans.swing.plaf.util.UIUtils.addInputMapsWithoutCtrlPageUpAndCtrlPageDown(Object[]) like other LaFs

Pull-Request for #114 and #126. Please try it out and merge it!

### CODE REVIEW

File: src/main/java/com/revivius/nb/darcula/DarculaLFCustoms.java [hunk 1]
```diff
@@ -472,6 +473,36 @@
...
+    }
+
+    /**
+     * Fixes https://github.com/Revivius/nb-darcula/issues/114
+     *
+     * @param result
+     * @return
+     */
+    private Object[] UIUtils_addInputMapsWithoutCtrlPageUpAndCtrlPageDown(Object[] result) {
+        /**
```
Comment (cowwoc) [inline src/main/java/com/revivius/nb/darcula/DarculaLFCustoms.java]: If this works, we should file a RFE against Netbeans to expose a public method.

**Comment by cowwoc** [comment]
@markiewb Take a look at http://hg.netbeans.org/main-silver/rev/e424830aeb3f. Can you apply the same fix here? It sounds like it might be more stable.

**Comment by markiewb** [comment]
@cowwoc: It sounds so, but it does not fix the issue. I tried it.

**Comment by AlexFalappa** [comment]
Please @Revivius merge this pull request and release a new version of NB Darcula, the broken tab navigation is really annoying for me and a lot of other people.

**Comment by Chris2011** [comment]
Anything new here?

**Comment by Revivius** [comment]
Done in 6fd446377913bc9b2f4891430307354997973702. Thank you @markiewb.

**Pull request closed without merging**